content/working_with/console/widgets/deploymentsView.md

```diff
@@ -0,0 +1,279 @@
+
+The columns shown in the table can be shown and hidden using the widget's
+configuration.
+
+<!--  TODO: screenshot -->
+
+The columns can show:
+
+1. The deployment status represented as an icon
+
+   - No icon - _good_
+   - Yellow spinner - _in progress_
+   - Red exclamation mark - _requires attention_
+
+   See [the Environments and Services section]({{< relref "#environments-and-services" >}})
+   for more information on deployment statuses.
+
+2. ID - the deployment ID
+
+   You can see the exact ID by hovering over the _ID_ button. From the popup
+   you can also copy the ID directly to your clipboard.
+
+3. Name - the deployment display name
+4. Blueprint Name - the name of the blueprint that this deployment is using
+5. Environment Type - the value of the `csys-env-type` label for that
+   deployment
+
+   Useful for differentiating deployments that use the same blueprint.
+
+6. Location - the name of the site the deployment is assigned to
+7. The counts and statuses of child deployments depending on their type
+   (environments or services)
+
+   See [the Environments and Services section]({{< relref "#environments-and-services" >}})
+   for more information on child deployments.
+
+   The counts and statuses in those columns are based on **all** child
+   deployments (it does not matter whether they are directly or indirectly
+   attached to the main deployment).
+
+Clicking a deployment row in the table will select it and affect the contents
+of the [Details pane]({{< relref "#details-pane" >}}).
+
+At the bottom of the table are controls that allow changing the number of
+deployments shown in a single table page, as well as moving to other pages in
+case there are more deployments that fit a single page.
+
+If there is a workflow in progress for a given environment, there is a progress
+bar right under the row in the table.
+
+### Sorting deployments
+
+Deployments in the table can be sorted by clicking a column's header.
+
+Clicking the column again will cycle between ascending and descending sort
+order.
+
+## Details pane
+
+The right side of the widget contains details about the currently selected
+deployment.
+
+In the header of the right side are:
+
+- the currently selected deployment name
+- buttons to drill-down to child environments or services
+
+  The buttons will be disabled when there are no child environments or
+  services.
+
+  See [the Environments and Services section]({{< relref "#environments-and-services" >}})
+  for more information on child deployments.
+
+- buttons to execute a workflow on that deployment or invoke some other
+  deployment action
+
+What follows are different deployment-related widgets divided into 3 tabs:
+
+1. Last Execution - information about the last workflow execution for that
+   deployment
+2. Deployment Info - general information about the deployment
+3. History - a list of workflow executions for that deployment and a list of
+   logs
+
+## Environments and Services
```

Comment:
I did not find any existing documentation for environments, so I described it directly here

Reply:
@ofercloudify / @mabuaisha Are we planning to have Environments / Services described somewhere in the user documentation? Or is it something which we will present through the Deployments View widget?